.github/ISSUE_TEMPLATE/---1-report-an-issue.md

```diff
@@ -27,6 +27,16 @@ assignees: ''
 ### Expected Outcome
 <!-- What outcome, for example query result, did you expect? -->
 
+### Failing Test Case / PR
```

Comment:
Can you expand `PR` -> `Pull Request (PR)` for new developers.

Reply:
done

Reply:
I wanted to add some more explanation of what a "failing test case" actually is, but I think that's for when we overhaul the contribution guide. I think it can be friendlier for new developers.

Reply:
I agree